addons/api/addon/generated/models/channel-recording.js

```diff
@@ -31,7 +31,7 @@ export default class GeneratedChannelRecordingModel extends BaseModel {
       'The total number of errors that occurred during the use of this Channel.',
     readOnly: true,
   })
-  errors;
+  errors_number;
```

Comment:
Just curious why this is changed? Was it always supposed to be `errors_number`?

Reply:
We are changing this because `errors` are a reserved word in ember and it was erroring out. more details [here](https://hashicorp.slack.com/archives/C012GTH1C4E/p1705429068221599)

Reply:
Thank you! I remember you mentioning this during standup now.